app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -99,7 +99,8 @@ def copy
       @new_activation_key.reload
       @new_activation_key.update_attributes!(
                                              :service_level => @activation_key.service_level,
-                                             :release_version => @activation_key.release_version
+                                             :release_version => @activation_key.release_version,
+                                             :auto_attach => @activation_key.auto_attach
```

Comment:
i feel like i should know this, but there is there a reason to not set all 3 of these attributes in ActivaitonKey#copy  ?